amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -8,492 +8,963 @@ input AMPLIFY {
   isDeleted: Boolean!
+  """Indicates whether the extension has been initialized."""
   isInitialized: Boolean!
+  """The version number of the extension."""
   version: Int!
 }
 
+"""Represents the current version of an entity in the system."""
 type CurrentVersion @model {
+  """Unique identifier for the CurrentVersion."""
   id: ID!
+  """The key used to look up the current version."""
   key: String! @index(name: "byKey", queryField: "getCurrentVersionByKey")
+  """The current version number."""
   version: Int!
 }
 
+"""Represents an action performed within a Colony."""
 type ColonyAction @model {
-  # Required fields common for all actions
+  """Unique identifier for the ColonyAction."""
   id: ID!
+  """The identifier of the Colony that the action belongs to."""
   colonyId: ID!
     @index(
       name: "byColony"
       queryField: "getActionsByColony"
       sortKeyFields: ["createdAt"]
     )
+  """The Colony that the action belongs to."""
   colony: Colony! @belongsTo(fields: ["colonyId"])
+  """The type of action performed."""
   type: ColonyActionType!
+  """The block number where the action was recorded."""
   blockNumber: Int!
+  """The timestamp when the action was created."""
   createdAt: AWSDateTime!
-  # Action type specific fields which might be null
-  initiatorAddress: ID # Amplify will automatically populate one of the following fields with related model if there is an initiator
+  """The Ethereum address of the action initiator. Can be a user, extension or colony."""
+  initiatorAddress: ID
+  # Amplify will automatically populate one of the following fields with related model if there is an initiator
+  """The User who initiated the action, if applicable."""
   initiatorUser: User @hasOne(fields: ["initiatorAddress"])
+  """The ColonyExtension that initiated the action, if applicable."""
   initiatorExtension: ColonyExtension @hasOne(fields: ["initiatorAddress"])
+  """The Colony that initiated the action, if applicable."""
   initiatorColony: Colony @hasOne(fields: ["initiatorAddress"])
+  """The Ethereum address of the action recipient, if applicable."""
   recipientAddress: ID
+  """The User who received the action, if applicable."""
   recipient: User @hasOne(fields: ["recipientAddress"])
+  """The amount involved in the action, if applicable."""
   amount: String
+  """The Ethereum address of the token involved in the action, if applicable."""
   tokenAddress: ID
+  """The Token involved in the action, if applicable."""
   token: Token @hasOne(fields: ["tokenAddress"])
+  """The source Domain identifier, if applicable."""
   fromDomainId: ID
+  """The source Domain of the action, if applicable."""
   fromDomain: Domain @hasOne(fields: ["fromDomainId"])
+  """The target Domain identifier, if applicable."""
   toDomainId: ID
+  """The target Domain of the action, if applicable."""
   toDomain: Domain @hasOne(fields: ["toDomainId"])
+  """ ??? Not sure what this is ???"""
   fundamentalChainId: Int
```

Comment:
Needs input

Reply:
I think @jakubcolony can also help here

Reply:
Copied over from the Dapp, it actually seems to only be relevant for motions, have you not come across this yet @willm30?

Reply:
Hmm, not yet. @rdig Do you know?